src/app/parcels/form/ParcelForm.tsx

```diff
@@ -159,6 +167,13 @@ const ParcelForm: React.FC<ParcelFormProps> = ({
     const [submitDisabled, setSubmitDisabled] = useState(false);
     const [modalIsOpen, setModalIsOpen] = useState<boolean>(false);
     const [clientDetails, setClientDetails] = useState<ExpandedClientData | null>(null);
+    const [collectionSlotsLabelsAndValues, setCollectionSlotsLabelsAndValues] =
+        useState<CollectionTimeSlotsLabelsAndValues>([
+            [
+                "Please select a collection centre for available collection slots",
+                "Collection centre not selected",
```

Comment:
Is this meant to be displayed if you open collection slots before selecting a centre? It just shows me a blank row (which I think I prefer - wouldn't want users selecting an error message. Could we put this as a validation error maybe?)

Reply:
Yes - just thought I'd cover all bases 🥲 The form currently does check that it has a value but only if it's previously null eg. if they already had a time for collection but not matching the slots it would still save. Should I edit this to remove the previous collection time so they have to select a new time?

Reply:
I'm not sure there is a good way to do this and make collection slots required.

When we added the packing slots - if the original packing slot was not available - there would be an error saying the original packing slot is no longer available, please select a different slot. If they didn't want to change it then they could leave the form.

I feel that either:
We keep it similar to packing slots to show an error message and tell them to select an available slot
or
Make collection slots not required?

Reply:
Currently the dropdown is set when a collection centre is selected.
If a collection centre is selected, I don't think its a good idea to put it in the drop down (feels a bit buggy to me)
We can put it as text in the card but they won't be able to save it because the dropdown is expecting a value unless we make it not required.

Reply:
I think matching what packing slot does is the best idea for consistency